compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEInsertLoopsForVectorization.cpp

```diff
@@ -74,21 +126,21 @@ class AMDAIEInsertLoopsForVectorizationPass
     // No outer dimensions to tile if fewer than 3 iterators.
     if (numIterators < 3) return failure();
 
+    bool isTensorType = isa<TensorType>(genericOp->getOperand(0).getType());
```

Comment:
Is this how tensor types are always derived? Wouldn't it be better to check the result?

Reply:
Good point. I'm now checking if at all there exists any result or not. Please take a look now.